lib/awful/wibar.lua

```diff
@@ -339,37 +339,19 @@ end
 -- You can also set the screen key with a screen number to attach the wibox.
 -- If not specified, the primary screen is assumed.
 -- @see wibox
--- @tparam[opt=nil] table arg
--- @tparam string arg.position The position.
--- @tparam string arg.stretch If the wibar need to be stretched to fill the screen.
--- @tparam integer arg.border_width Border width.
--- @tparam string arg.border_color Border color.
--- @tparam boolean arg.ontop On top of other windows.
--- @tparam string arg.cursor The mouse cursor.
--- @tparam boolean arg.visible Visibility.
--- @tparam number arg.opacity The wibar's opacity, between 0 and 1.
--- @tparam string arg.type The window type (desktop, normal, dock, …).
--- @tparam integer arg.x The x coordinates.
--- @tparam integer arg.y The y coordinates.
--- @tparam integer arg.width The wibar's width.
--- @tparam integer arg.height The wibar's height.
--- @tparam screen arg.screen The wibox screen.
--- @tparam wibox.widget arg.widget The widget that the wibox displays.
--- @param arg.shape_bounding The wibox’s bounding shape as a (native) cairo surface.
--- @param arg.shape_clip The wibox’s clip shape as a (native) cairo surface.
--- @param arg.shape_input The wibox’s input shape as a (native) cairo surface.
--- @tparam color arg.bg The wibar's background.
--- @tparam surface arg.bgimage The background image of the drawable.
--- @tparam color arg.fg The wibar's foreground (text) color.
+-- @tparam[opt=nil] table args
+-- @tparam string args.position The position.
+-- @tparam string args.stretch If the wibar need to be stretched to fill the screen.
+--@DOC_wibox_constructor_COMMON@
 -- @return The new wibar
 -- @function awful.wibar
-function awfulwibar.new(arg)
-    arg = arg or {}
-    local position = arg.position or "top"
+function awfulwibar.new(args)
+    args = args or {}
```

Comment:
This seems a lot like an (almost) unrelated change that should be in its own commit, but okay, whatever....